chriremm/Fibonacci-Flask-Heroku
# Feature: Add Bootstrap UI and Result Modal to Fibonacci Calculator

## Overview
This pull request introduces significant updates to the UI of the Fibonacci Calculator. It incorporates Bootstrap to improve the overall aesthetics and user experience. Additionally, the result display has been shifted from a simple text element to a Bootstrap modal for a cleaner and more modern presentation.

## Changes Made
- Integrated Bootstrap 4.3.1 into the application.
- Updated the form layout using Bootstrap Grid system to enhance responsiveness.
- Added a modal for displaying the Fibonacci calculation result, which is triggered upon form submission.
- Modified the JavaScript fetch logic to display results in the modal instead of inline text.
- Refactored the HTML structure to utilize Bootstrap's styling classes effectively.

## How to Test
1. Pull the changes from the feature branch into your local environment.
2. Run the Flask application.
3. Navigate to the main page and submit a number to calculate its Fibonacci number.
4. Verify that the result is displayed in a modal and the UI elements are responsive across different screen sizes.

Looking forward to feedback on these changes. Thank you for considering this pull request.
